apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisFiltersPopper.vue

```diff
@@ -37,7 +38,12 @@ const storeState = reactive({
 });
 const state = reactive({
     loading: true,
-    enabledFilters: computed<MenuItem[]>(() => costAnalysisPageStore.defaultGroupByItems.filter((d) => costAnalysisPageState.enabledFiltersProperties?.includes(d.name))),
+    enabledFilters: computed<MenuItem[]>(() => {
+        if (costAnalysisPageStore.selectedQueryId === DYNAMIC_COST_QUERY_SET_PARAMS) {
+            return costAnalysisPageStore.defaultGroupByItems;
+        }
+        return costAnalysisPageStore.defaultGroupByItems.filter((d) => costAnalysisPageState.enabledFiltersProperties?.includes(d.name));
+    }),
```

Comment:
description: for here

![스크린샷 2023-09-14 오후 1 00 26](https://github.com/cloudforet-io/console/assets/18563857/01838aa3-be82-4d55-8bd1-b5e6f8b4a25d)